dspace-api/src/main/java/org/dspace/content/service/DSpaceObjectService.java

```diff
@@ -52,6 +52,10 @@
             = new MetadataFieldName(DC, "format");
     public static final MetadataFieldName MD_SOURCE
             = new MetadataFieldName(DC, "source");
+    /*
+     * Field used to sort community and collection lists at solr
+     */
+    public static final String COMMUNITIES_AND_COLLECTIONS_SORT_FIELD = "dc.title_sort";
```

Comment:
Small thing, but this appears to only be used for *Collection* sorting at this time.  So, it's name is misleading, and it probably would be better added to `CollectionService`.

I'd recommend moving this to `CollectionService` and renaming to simply `CollectionService.COLLECTIONS_SORT_FIELD` (or possibly even `CollectionService.SOLR_SORT_FIELD`).